domains/runtime/evm/src/lib.rs

```diff
@@ -693,7 +692,10 @@ pub fn extract_signer(
     extrinsics
         .into_iter()
         .map(|extrinsic| {
-            let maybe_signer = extract_signer_inner(&extrinsic, &lookup);
+            let maybe_signer =
```

Comment:
As you have mentioned before, shouldn't we be returning the error ?

Reply:
I dont think so, if we are returning an error when extracting a signer, then it should be handled on the client side and error out accordingly IMO

Reply:
@vedhavyas No, I agree. What I am saying is this PR may not be the place to do it as changes will propagate through areas of code unrelated to this PR. How about I create an issue and open a separate PR for this?